lang/en.json

```diff
@@ -461,10 +495,6 @@
             "label": "Critical Threshold",
             "abbr": "CRIT"
           },
-          "blight": {
```

Comment:
I think that this "Blight" translation should be kept, as this is a new change we introduced for some weapons present in the Flowers of Algorab. Removing that will break the weapon sheet.